solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -566,12 +577,21 @@ contract RandomBeacon is Ownable {
     ///         It removes a candidate group that was previously registered with
     ///         the DKG result submission.
     function challengeDkgResult() external {
-        // TODO: Determine parameters required for DKG result challenges.
-        dkg.challengeResult();
+        bytes32 resultHash = dkg.submittedResultHash;
```

Comment:
`resultHash` is only used in the event, do we need to cache it this way? Shouldn't we inline the call where we emit `DkgResultMaliciousSlashed`?

Reply:
Yep, we need it. Some cleanup is done within `dkg.challengeResult()`.

Reply:
Can we have challengeResult returning the hash? I think this is cleaner - we do not have to look inside about what information DKG library stores and when it is cleared up.

Reply:
Sure: https://github.com/keep-network/keep-core/pull/2695/commits/7c9f436788770bf52471df6721aa35ce6aa6f937